src/bin/initialize-login-theme.ts

```
@@ -0,0 +1,323 @@
+        const moduleName = "@keycloakify/login-ui";
+
+        const latestVersion = await getModuleLatestVersion({ moduleName });
+
+        (parsedPackageJson.dependencies ??= {})[moduleName] = `~${latestVersion}`;
+
+        if (parsedPackageJson.devDependencies !== undefined) {
+            delete parsedPackageJson.devDependencies[moduleName];
+        }
+    }
+
+    install_stories: {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)